src/table/components/head/__tests__/HeadCellMenu.spec.tsx

```diff
@@ -1,21 +1,29 @@
+/* eslint-disable no-underscore-dangle */
 import React from 'react';
 import { stardust } from '@nebula.js/stardust';
 import { render, fireEvent, screen, waitFor, waitForElementToBeRemoved } from '@testing-library/react';
-import { TableLayout } from '../../../../types';
+import { Column, TableLayout } from '../../../../types';
 import HeadCellMenu from '../HeadCellMenu';
 import TestWithProviders from '../../../../__test__/test-with-providers';
 
+type EmbedWithDoNotUse = stardust.Embed & {
```

Comment:
DIdn't think of this before. I think we should do this extending as for `ExtendedTranslator` etc. In the index file, we cast to this new type `ExtendedEmbed` and then we use that type throughout  the code

Reply:
I would suggest let's do it in another PR and not pollute this one with that change, seems like there are lots of them.